formatter/response_formatters.py

```diff
@@ -34,8 +34,9 @@ def get_exam_formatter(response: pb.ExaminationSchedule) -> str:
         msg += f"{exam.course.code} \n"
         msg += f"{exam.course.name} \n"
         msg += f"{date.strftime('%d %b %Y')} \n"
+        msg += f"{date.strftime('%H:%M')} \n"
+        msg += f"{exam.mode} \n"
         try:
-            msg += f"{exam.mode} \n"
             msg += f"{exam.location} \n\n"
         except:
             msg += "\n"
```

Comment:
```suggestion
            msg += ""
```
No need to create a newline if exam.location doesn't exist as it's simply creating a space that isn't needed
<img width="268" alt="Screenshot 2023-11-09 at 12 53 33 AM" src="https://github.com/asetalias/amibot-tg/assets/14011425/0b4e0381-9e49-4cdd-9b93-1dc933a1b000">